spark/src/test/resources/delta/table-with-dv-special-char/_delta_log/00000000000000000000.json

```diff
@@ -0,0 +1,4 @@
+{"commitInfo":{"timestamp":1708950820866,"operation":"WRITE","operationParameters":{"mode":"ErrorIfExists","partitionBy":"[]"},"isolationLevel":"Serializable","isBlindAppend":true,"operationMetrics":{"numFiles":"1","numOutputRows":"10","numOutputBytes":"539"},"engineInfo":"Apache-Spark/3.5.0 Delta-Lake/3.1.0-SNAPSHOT","txnId":"1586fe39-668f-48c6-ad0b-af4a10b38f22"}}
+{"metaData":{"id":"testId","format":{"provider":"parquet","options":{}},"schemaString":"{\"type\":\"struct\",\"fields\":[{\"name\":\"id\",\"type\":\"long\",\"nullable\":true,\"metadata\":{}}]}","partitionColumns":[],"configuration":{"delta.enableDeletionVectors":"true"},"createdTime":1708950819168}}
+{"protocol":{"minReaderVersion":3,"minWriterVersion":7,"readerFeatures":["deletionVectors"],"writerFeatures":["deletionVectors"]}}
+{"add":{"path":"part-00000-8d24f407-08d3-49ab-9d1c-f7f6c129e882-c000.snappy.parquet","partitionValues":{},"size":539,"modificationTime":1708950820783,"dataChange":true,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":0},\"maxValues\":{\"id\":9},\"nullCount\":{\"id\":0},\"tightBounds\":false}","deletionVector":{"storageType":"p","pathOrInlineDv":"file:{{FOLDER_WITH_SPECIAL_CHAR}}/deletion_vector_67bc892e-2979-4760-a78c-856aba806564.bin","offset":1,"sizeInBytes":42,"cardinality":5}}}
```

Comment:
If it's not too much work, maybe we should add a test for storageType "u" , where the parent directory is the one that needs special handling. Or is that handled by the other tests?

Reply:
Yes, that case will be handled by the next PR, in which we add special characters to `withTempDir`-generated folders.